ooyala_player/ooyala_player.py

```diff
@@ -491,7 +327,7 @@ class OoyalaPlayerLightChildBlock(OoyalaPlayerMixin, LightChild):
         display_name=_("Content Id"),
         help=_("Identifier for the Content Id."),
         scope=LCScope.content,
-        default='RpOGxhMTE6p6DkTB8MBGtKN6v0_A_BdQ'
+        default='6110589153001'
```

Comment:
hoping this is not one of our video id. :)

Reply:
It is one of our video but it's [mcka requiremen](https://edx-wiki.atlassian.net/browse/MCKIN-13055) to use this placeholder video. Other option could be putting it in configurable settings